src/ConnectedMode.UnitTests/Shared/SharedBindingConfigFileProviderTests.cs

```diff
@@ -63,7 +62,7 @@ public void ReadSharedBindingConfigFile_SCConfig_Reads()
         {
             string configFileContent = @"{""SonarCloudOrganization"":""Some Organisation"",""ProjectKey"":""projectKey""}";
             string filePath = "Some Path";
-            var uri = new Uri(SharedBindingConfigFileProvider.SonarCloudUri);
+            var uri = SharedBindingConfigFileProvider.SonarCloudUri;
```

Comment:
do we really need this assignment anymore?

Reply:
No, I missed it